mycli/myclirc

```diff
@@ -86,6 +86,11 @@ Token.Menu.Completions.MultiColumnMeta = 'bg:#aaffff #000000'
 Token.Menu.Completions.ProgressButton = 'bg:#003333'
 Token.Menu.Completions.ProgressBar = 'bg:#00aaaa'
 
+# Query results
+Token.Output.Header = 'bold'
+Token.Output.OddRow = ''
+Token.Output.EvenRow = ''
```

Comment:
It's a bit hard to think of anything that would work well for both light and dark terminal theme though. Easy to end up with invisible text.